### ISSUE TITLE
Help text is ugly

### ISSUE THREAD

**Issue by georgefst** [fourmolu/fourmolu#31]
Perhaps my fault for choosing some rather long field names... But really this is an `optparse-applicative` issue: https://github.com/pcapriotti/optparse-applicative/issues/406

```
  --start-line START       Start line of the region to format (starts from 1)
  --end-line END           End line of the region to format (inclusive)
  --indentation WIDTH      Number of spaces per indentation step (default 4)
  --comma-style STYLE      How to place commas in multi-line lists, records etc: 'leading' (default) or 'trailing'
  --indent-wheres BOOL     Whether to indent 'where' bindings past the preceding body (rather than half-indenting the
                           'where' keyword) (default 'false')
  --record-brace-space BOOL
                           Whether to leave a space before an opening record brace (default 'false')
  --diff-friendly-import-export BOOL
                           Whether to make use of extra commas in import/export lists (as opposed to Ormolu's style)
                           (default 'true')
  --preserve-spacing BOOL  Give the programmer more choice on where to insert blank lines (default 'true')
  --haddock-style STYLE    How to print Haddock comments (default 'multi-line')
  FILE                     Haskell source files to format or stdin (default)
```

**Comment by georgefst** [comment]
> But really this is an optparse-applicative issue

Very quick response over there. Fixed and should be in the next release.

**Comment by brandonchinn178** [comment]
ref:
* https://github.com/pcapriotti/optparse-applicative/issues/406
* https://github.com/pcapriotti/optparse-applicative/issues/258
* https://github.com/pcapriotti/optparse-applicative/pull/407

Fixed in optparse-applicative 0.17

**Comment by brandonchinn178** [comment]
Sorry, it seems like optparse-applicative 0.17 will just make it configurable. Reopening to fix the help text after optparse-applicative 0.17 is released
